test/integration/outreach/outreach.js

```diff
@@ -45,7 +45,7 @@ context('Outreach', function() {
     cy
       .get('.js-phone')
       .clear()
-      .type('+18887771234');
+      .type('8887771234');
```

Comment:
By removing the `+1` here, we can test that the `+1` formatting is sent in the api request whether the user types it or not.